src/chronometer.js

```diff
@@ -1,33 +1,59 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+  this.currentTime=0;
+  this.intervalId=null;
   }
-
+/////////////////////////////////////////////////////////////////
   start(printTimeCallback) {
-    // ... your code goes here
-  }
 
-  getMinutes() {
-    // ... your code goes here
+let contadorM=0;
+let contadorS=0;
+setInterval(()=> {
+   this.currentTime++
+    if(contadorS<3){
```

Comment:
-  Si existe el argumento, debes invocar `printTimeCallback` dentro de tu intervalo para que se ejecute cada segundo.
- En esta función no necesitas agregar contadores, `this.currentTime++` ya es tu contador que se incrementa 1 cada vez que sucede tu intervalo (cada segundo).